Python/Templates/IronPython/ProjectTemplates/Python/ConsoleProject/ConsoleProject.pyproj

```diff
@@ -24,7 +24,7 @@
     <None Include="$safeprojectname$.py" />
   </ItemGroup>
   <ItemGroup>
-    <InterpreterReference Include="{80659ab7-4d53-4e0c-8588-a766116cbd46}\2.7" />
+    <InterpreterReference Include="IronPython|2.7 32-bit" />
```

Comment:
Can we use "-32" as the suffix for 32-bit rather than having a space in there? It _should_ be fine, but I've learned to be wary of spaces in identifiers.